components/demo/demo-page.js

```diff
@@ -86,6 +112,13 @@ class DemoPage extends LitElement {
 			this._noScroll = true;
 		}
 	}
+
+	_handleLanguageChange(e) {
+		const newLanguageCode = e.target[e.target.selectedIndex].value;
+		document.documentElement.dir = newLanguageCode === 'ar-sa' ? 'rtl' : 'ltr';
```

Comment:
So now to test RTL visually, we'll need to explicitly select `ar-sa`. Sounds good, might be worth calling out on our Slack or something though 😄.